### DISCUSSION TITLE
support for latest version of IDE?

### DISCUSSION THREAD

**Discussion by MSerj** [AVeenstra/RunAnything#5] [General]
Any plans to add support for latest version of IDE?
![image](https://user-images.githubusercontent.com/7062888/163176790-d64f97ae-91b4-4ac2-a389-bf39637fcdac.png)

**Comment by AVeenstra** [comment]
I will have time this weekend, so it will probably be updated on Monday. Sorry for the delay.

**Reply by MSerj** [reply]
Very good news, no need to apologise. Thank you for the plugin ❤

**Reply by AVeenstra** [reply]
Looks like JetBrains already approved the new version, so it should work again. Let me know if it's not working!

**Comment by MSerj** [comment]
@AVeenstra hey hey, sorry to bother you again, **enjoying your plugin** but i have the same issue with new version of WS. Any plans for supporting new version aswell?
